File: app/src/main/java/com/example/aluraviagens/ui/activity/ListasPacotesActivity.java
package com.example.aluraviagens.ui.activity;

import android.app.Activity;
import android.os.Bundle;
import android.widget.ListView;

import com.example.aluraviagens.R;
import PacoteDAO.PacoteDAO;
import com.example.aluraviagens.ui.adapter.ListaPacotesAdapter;
import model.Pacote;

import java.util.List;

public class ListasPacotesActivity extends Activity {

    public static final String TITULO_APPBAR = "Pacotes";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_lista_pacotes);
        setTitle(TITULO_APPBAR);
        configuraLista();
    }

    private void configuraLista() {
        ListView listaDePacotes = findViewById(R.id.list_view_pacotes);
        List<Pacote> pacotes = new PacoteDAO().lista();
        listaDePacotes.setAdapter(new ListaPacotesAdapter(pacotes, this));
    }

}

File: app/src/main/java/com/example/aluraviagens/ui/activity/ListasPacotesActivity.kt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ListView
import com.example.aluraviagens.R
import PacoteDAO.PacoteDAO
import com.example.aluraviagens.ui.adapter.ListaPacotesAdapter
import model.Pacote

class ListasPacotesActivity : AppCompatActivity() {

    private val mListaDAO: List<Pacote> = PacoteDAO().lista()
    private lateinit var mAdapter: ListaPacotesAdapter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_lista_pacotes)

        mAdapter = ListaPacotesAdapter(mListaDAO, this)

        var listaDePacotes: ListView = findViewById(R.id.list_view_pacotes)
        listaDePacotes.adapter = mAdapter

    }
}